bot/tests/sending_videos/test_send_clip.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+
+from bot.tests.base_test import BaseTest
+
+
+class TestSendClipCommand(BaseTest):
+
+    @pytest.mark.quick
+    def test_send_existing_clip(self):
+        self.send_command('/klip geniusz')
+        self.send_command('/zapisz klip1')
+
+        send_response = self.send_command('/wyslij klip1')
+        self.assert_video_matches(send_response, 'geniusz_clip_sent.mp4')
```

Comment:
jezeli to ma zalezec od nazwy cytatu to tez wynies do zmiennej i podstaw w templatke, porzadek musi byc

Reply:
do wszystkiego ofc

Reply:
no po prostu zebys zrobil var

name="klip1"
f"/zapisz {name}"